Source/UI/BillingForm/FramesFactory.swift

```diff
@@ -33,7 +33,24 @@ public enum FramesFactory {
         if #available(iOS 13.0, *) {
             viewController.isModalInPresentation = true
         }
-        return UINavigationController(rootViewController: viewController)
+        guard let sender = sender else {
+            return UINavigationController(rootViewController: viewController)
+        }
+        let navigationController = UINavigationController(rootViewController: viewController)
+        navigationController.navigationBar.tintColor = sender.navigationBar.tintColor
+
+        if #available(iOS 13.0, *) {
+            navigationController.navigationBar.standardAppearance = sender.navigationBar.standardAppearance
+        } else {
+            let backgroundColor = sender.navigationBar.backgroundColor
+            navigationController.navigationBar.backgroundColor = backgroundColor
+            navigationController.navigationBar.barTintColor = backgroundColor
+            navigationController.navigationBar.shadowImage = sender.navigationBar.shadowImage
+            navigationController.navigationBar.titleTextAttributes = [ .foregroundColor: sender.navigationBar.tintColor ?? .black]
+            navigationController.navigationBar.isTranslucent = true
+        }
+
+        return navigationController
```

Comment:
I'm not certain why this customisation is needed now? Was this a hidden "bug" / lack of consistency?

I feel tempted to move this inside an extension from `UINavigationController`:
```swift
extension UINavigationController {

    convenience init(copyStyleFrom originalNavController: UINavigationController, rootViewController: UIViewController) {
        /* Create the new bar and pass customisation */
    }
}
```

Then the code here becomes:
```swift
guard let presentingNavigationController = presentingNavigationController else {
    return UINavigationController(rootViewController: viewController)
}
return UINavigationController(copyStyleFrom: presentingNavigationController, rootViewController: viewController)
```

we can also unit test the transformation to ensure our expected behaviour is reinforced and confirmed